Product/Production/Services/AuditRepositoryCore/src/main/java/gov/hhs/fha/nhinc/auditrepository/nhinc/AuditRepositoryOrchImpl.java

```diff
@@ -196,6 +173,35 @@ public FindCommunitiesAndAuditEventsResponseType findAudit(FindAuditEventsType q
         return auditEvents;
     }
 
+    protected final AuditRepositoryRecord createDBAuditObj(LogEventSecureRequestType mess, AssertionType assertion) {
+        AuditRepositoryRecord auditRec = new AuditRepositoryRecord();
+        EventIdentificationType eventIdentification = mess.getAuditMessage().getEventIdentification();
```

Comment:
These are available in LogEventRequestType schema so they can be accessed from mess here.